example/ios/AppSpectorPluginTests/AppSpectorPluginTests.m

```diff
@@ -187,4 +187,29 @@ - (void)testHandlerRejectsUnknownCalls {
     [self waitForExpectations:@[e] timeout:0.1];
 }
 
+- (void)testHandlerExtractsEnvFlagFromArgs {
+    XCTestExpectation *e = [self expectationWithDescription:@""];
+    
+    FlutterMethodCall *call = [FlutterMethodCall methodCallWithMethodName:@"run" arguments:@{ @"apiKey" : @"DEADBEEF",
+                                                                                              @"enabledMonitors" : @[],
+                                                                                              @"metadata" : @{ @"APPSPECTOR_CHECK_STORE_ENVIRONMENT" : @(NO) }
```

Comment:
The metadata is receiving "string":"string" pairs in the public API. Is it correct?